packages/main/src/Menu.ts

```diff
@@ -253,7 +253,9 @@ class Menu extends UI5Element {
 	loadingDelay!: number;
 
 	/**
-	 * Defines the ID or DOM Reference of the element that the menu is shown at
+	 * Defines the ID or DOM Reference of the element that the menu is shown at.
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to show the popover at.
```

Comment:
same:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.